extensions/likes/js/src/forum/addLikesTabToUserProfile.tsx

```diff
@@ -0,0 +1,21 @@
+import { extend } from 'flarum/common/extend';
+import app from 'flarum/forum/app';
+import UserPage from 'flarum/forum/components/UserPage';
+import LinkButton from 'flarum/common/components/LinkButton';
+import LikesUserPage from './components/LikesUserPage';
+import ItemList from 'flarum/common/utils/ItemList';
+import type Mithril from 'mithril';
+
+export default function addLikesTabToUserProfile() {
+  app.routes['user.likes'] = { path: '/u/:username/likes', component: LikesUserPage };
+  extend(UserPage.prototype, 'navItems', function (items: ItemList<Mithril.Children>) {
+    const user = this.user;
+    items.add(
+      'likes',
+      <LinkButton href={app.route('user.likes', { username: user.slug() })} name="likes" icon="far fa-thumbs-up">
```

Comment:
Don't think this has any effect (core seems to use it for mentions, but not for any other links -- not sure what's going on there).

```suggestion
      <LinkButton href={app.route('user.likes', { username: user.slug() })} icon="far fa-thumbs-up">
```

Reply:
Yeah, I thought it a little strange too, but as mentions had it..... 🤷‍♂️